tests/integration/test_status.py

```diff
@@ -0,0 +1,1110 @@
+                ),
+            ],
+            1,
+        ),
+        (
+            [
+                PullRequest(
+                    id=1,
+                    title="PR1",
+                    url="URL1",
+                    status=PRStatus.all,
+                    description="",
+                    author="",
+                    source_branch="",
+                    target_branch="",
+                    created=datetime.datetime.fromtimestamp(int(0)),
+                )
+            ],
+            1,
+        ),
+        (
+            [
+                PullRequest(
+                    id=1,
+                    title="PR1",
+                    url="URL1",
+                    status=PRStatus.all,
+                    description="",
+                    author="",
+                    source_branch="",
+                    target_branch="",
+                    created=datetime.datetime.fromtimestamp(int(0)),
+                ),
+                PullRequest(
+                    id=2,
+                    title="PR2",
+                    url="URL2",
+                    status=PRStatus.all,
+                    description="",
+                    author="",
+                    source_branch="",
+                    target_branch="",
+                    created=datetime.datetime.fromtimestamp(int(0)),
+                ),
+                PullRequest(
+                    id=3,
+                    title="PR3",
+                    url="URL3",
+                    status=PRStatus.all,
+                    description="",
+                    author="",
+                    source_branch="",
+                    target_branch="",
+                    created=datetime.datetime.fromtimestamp(int(0)),
+                ),
+                PullRequest(
+                    id=4,
+                    title="PR4",
+                    url="URL4",
+                    status=PRStatus.all,
+                    description="",
+                    author="",
+                    source_branch="",
+                    target_branch="",
+                    created=datetime.datetime.fromtimestamp(int(0)),
+                ),
+            ],
+            2,
+        ),
+    ],
+)
+def test_downstream_pr(upstream_n_distgit, pr_list, number_prs):
+    u, d = upstream_n_distgit
+
+    c = get_test_config()
+    pc = get_local_package_config(str(u))
+    pc.downstream_project_url = str(d)
+    pc.upstream_project_url = str(u)
+    dg = DistGit(c, pc)
+    pc = get_local_package_config(str(u))
+    status = Status(c, pc, u, dg)
+    flexmock(
+        PagureProject,
+        get_git_urls=lambda: {"git": "foo.git"},
+        fork_create=lambda: None,
+        get_fork=lambda: PagureProject("", "", PagureService()),
+        get_pr_list=pr_list,
+    )
+    assert status
+    table = status.get_downstream_prs(number_prs)
+    assert table
+    assert len(table) == number_prs
+
+
+@pytest.fixture()
+def bodhi_response_updates():
```

Comment:
I am not sure if we need all 800 lines. (At least we should place it in another file.)

Reply:
Yeah, I was thinking about that. E.g. create a new file called `tests/bodhi.py` and place there all bodhi fixtures. Including `tests/integration/test_create_update`

Reply:
Some developers need full response from a command, like bodhi, and some of them need only important stuff, like in our GitLab. Will send over PM.

Reply:
+1 for adding the bodhi giga-json into a separate file

Reply:
moved into `tests/integration/bodhi_status_updates.py`